### PR TITLE
Update version in st2common/__init__ and st2client/__init__

### PR DESCRIPTION
**Pull request by manasdk**

### CODE REVIEW

**Comment by DoriftoShoes** [comment]
+1

File: packs/st2cd/actions/git_branch.sh [hunk 1]
```diff
@@ -6,14 +6,22 @@ DATE=`date +%s`
...
 OUT=`$GIT checkout ${BASE_BRANCH} -q > $OUTPUT && $GIT pull origin ${BASE_BRANCH} -q > $OUTPUT && $GIT checkout -b ${BRANCH} -q`
 if [[ $? == 0 ]]
 then
-  VERSIONOUT=`sed -i -e "s/\(__version__ = \).*/\1'${VERSION}'/" ${FILE}`
+  VERSIONOUT=`sed -i -e "s/\(__version__ = \).*/\1'${VERSION}'/" ${ST2COMMON_INIT}`
+  if [[ $? == 0 ]]
+  then
+    VERSIONOUT=`sed -i -e "s/\(__version__ = \).*/\1'${VERSION}'/" ${ST2CLIENT_INIT}`
+  else
+    echo ${OUTPUT}
```
Comment (lakshmi-kannan) [inline packs/st2cd/actions/git_branch.sh]: echo "Failed setting version in ${ST2COMMON_INIT}"

Comment (DoriftoShoes) [inline packs/st2cd/actions/git_branch.sh]: Why do we echo the output location here and elsewhere we cat it, then remove it?

Comment (manasdk) [inline packs/st2cd/actions/git_branch.sh]: Dude I have no clue what we are outputting anywhere in this file. Its all strange to me. So tell me what you want and I will do that :P

Comment (DoriftoShoes) [inline packs/st2cd/actions/git_branch.sh]: cat ${OUTPUT}
rm ${OUTPUT}
exit 2

Comment (DoriftoShoes) [inline packs/st2cd/actions/git_branch.sh]: Or cat ${VERSIONOUT} in there somewhere too.

Comment (manasdk) [inline packs/st2cd/actions/git_branch.sh]: Done.

**Pull request merged**